docs/_plugins/importMap.cjs

```diff
@@ -48,56 +58,35 @@ async function getCachedImportMap({
 
       const { Generator } = await import('@jspm/generator');
 
+      const nothing = Symbol();
+      const providers = {
+        '@patternfly': 'nodemodules',
+        ...Object.fromEntries(localPackages?.map(packageName =>
+          packageName.match(/@(rhds|patternfly)/) ? [nothing] : [packageName, 'nodemodules']) ?? []),
+      };
```

Comment:
More of a missing understanding thing.  I was trying to figure out if `nodemodules` was a typo, started looking at docs for jspm generator, couldn't find an explanation for why `nodemodules` was written this way vs `node_modules` as typical.

Is this due to the file name of the provider also being compacted? 
https://github.com/jspm/generator/blob/63ad57d21e5aeb2a6678b9a4a6da455fa75648cc/src/providers/nodemodules.ts

Have I dug to far? 🙃

Reply:
Also not blocking feel free to resolve with no explanation.

Reply:
yeah im' not really sure why, this is just what Guy Bedford called it :shrug: